pages/contribute/rule-feedback.md

```diff
@@ -0,0 +1,49 @@
+---
+title: Give feedback on rules
+---
+
+Feedback on rules is encouraged on our open [GitHub repository](https://github.com/act-rules/act-rules.github.io), so you don't even have to be a member of the Community Group to contribute in this way.
+
+If you "watch" the repository, you will get notifications when changes are happening.
+
+## Feedback for ideas and early drafts (Issues on GitHub)
+
+Rules and Definitions (shared terms across rules) that are GitHub Issues can be everything from just a title to a quite finished rule. The issues will be prefixed with either "Rule:" (or possibly "New Rule:") or "Algorithm".
+
+The top comment on an issue should hold the updated rule or algorithm, and underneath there will be a thread of comments and a history of actions related to the issue.
+
+Please feel free to add your own comments to issues.
+
+Start giving feedback on issues:
+
+- See the [ideas and early drafts that might need input](https://github.com/act-rules/act-rules.github.io/issues)
```

Comment:
Should we filter that on labels such as "rule idea" or "Should we proceed?" or some other?
Many issues are problem with existing rules and not ideas/early draft, and the feedback to give is quite different…

Reply:
That would be good, if there is a generic filter for "ideas and early drafts", which is what this section covers. It would need to include definition drafts as well as rule ideas and drafts, and anything similar.